apps/demo/translations/src/en/buttons.json

```diff
@@ -0,0 +1,3 @@
+{
+  "start": "Start"
+}
```

Comment:
newline, I tend to review my own pr before I do a PR

also, vscode or any other editor will do newliines for you

Reply:
I have added a new line now